daffodil-cli/src/main/scala/org/apache/daffodil/InfosetTypes.scala

```diff
@@ -643,7 +653,12 @@ case class SAXInfosetHandler(dataProcessor: DataProcessor, forPerformance: Boole
     def replay(h: ContentHandler): Unit = h.startDocument()
   }
 
-  case class SaxEventStartElement(uri: String, localName: String, qName: String, atts: Attributes) extends SaxEvent {
+  case class SaxEventStartElement(
+    uri: String,
+    localName: String,
+    qName: String,
+    atts: Attributes)
+    extends SaxEvent {
     def replay(h: ContentHandler): Unit = h.startElement(uri, localName, qName, atts)
   }
```

Comment:
> I've enabled rewrite.trailingCommas.style = always.

It looks like scalafmt doesn't adds a trailing comma to the last constructor/method parameter despite what `rewrite.trailingCommas.style=always` says.  That's a shame because now the extends clause and the def statement have nothing visually separating them anymore.  If scalafmt had added the trailing comma to the last parameter, the ) and the extend clause would be on the same line and there would be a clear visual separation:

```scala
  case class SaxEventStartElement(
    uri: String,
    localName: String,
    qName: String,
    atts: Attributes,
  ) extends SaxEvent {
    def replay(h: ContentHandler): Unit = h.startElement(uri, localName, qName, atts)
  }
```